tox.ini

```diff
@@ -1,20 +1,20 @@
 [tox]
-envlist = py{26,27,py,33,34,35}-{test,uwsgi,stylecheck}
+envlist = py{26,27,py,33,34,35}-{normal,uwsgi,stylecheck}
 
 [testenv]
 passenv = LANG
 deps=
 # General
-    pyopenssl
-    greenlet
-    pytest
-    pytest-xprocess
-    redis
-    requests
-    watchdog
-
-    uwsgi: uwsgi
+    normal: pyopenssl
```

Comment:
not sure off-hand - on pytest itself we generally use a setup with a completely differently named testenv for linting - but we dont lint for all python versions as well, will need a bit to investigate